lib/plenario2/changesets/data_set_field_changesets.ex

```diff
@@ -1,6 +1,13 @@
 defmodule Plenario2.Changesets.DataSetFieldChangesets do
   import Ecto.Changeset
 
+  def create(struct) do
```

Comment:
I'm not sure I like this. This is very specific to the web controller and provides no advantage over calling a bare `create(%DataSetField{}, %{})`.

Syntactic sugar is good and all, but this is the core library and I'm always weary of adding code for one off uses.